nautobot/core/management/commands/generate_test_data.py

```diff
@@ -156,6 +157,11 @@ def _create_batch(some_factory, count, description="", **kwargs):
         _create_batch(RoleFactory, 20)
         populate_status_choices(verbosity=0, using=db_name)
         _create_batch(StatusFactory, 10)
+        from nautobot.dcim.models import VirtualDeviceContext
+        from nautobot.extras.models import Status
+
+        st = Status.objects.create(name="Sample")
+        st.content_types.add(ContentType.objects.get_for_model(VirtualDeviceContext))
```

Comment:
remove;
after fix: No status exists for VDC model causing failing vdc factory initialisation